internal/drivers/noop/driver.go

```diff
@@ -0,0 +1,99 @@
+package noop
+
+import (
+	"context"
+	"time"
+
+	"github.com/drone-runners/drone-runner-aws/internal/drivers"
+	"github.com/drone-runners/drone-runner-aws/internal/lehelper"
+	"github.com/drone-runners/drone-runner-aws/types"
+	"github.com/google/uuid"
+)
+
+type config struct {
+	rootDir           string
+	hibernate         bool
+	hibernateWaitSecs int
+	startWaitSecs     int
+	createWaitSecs    int
+	destroyWaitSecs   int
+	tagWaitSecs       int
+	leIP              string
+}
+
+func New(opts ...Option) (drivers.Driver, error) {
+	p := new(config)
+	for _, opt := range opts {
+		opt(p)
+	}
+
+	p.hibernateWaitSecs = 5
```

Comment:
could be made configurable (from yaml) with default values? feel free to ignore if you think it's unnecessary

Reply:
not required for now